src/Avalonia.Base/Input/Pointer.cs

```diff
@@ -79,6 +81,12 @@ private void OnCaptureDetached(object? sender, VisualTreeAttachmentEventArgs e)
         /// </summary>
         internal GestureRecognizer? CapturedGestureRecognizer { get; private set; }
 
+        /// <summary>
+        /// The previously captured element. This is set when a <see cref="GestureRecognizer"/> captures
+        /// the pointer.
+        /// </summary>
+        internal IInputElement? PreviousCaptured { get; private set; }
```

Comment:
Need to be careful here to not memory leak anything. Leak test with simulated gestures would be nice as well.